docs/frequently-asked-questions.md

```diff
@@ -133,21 +133,21 @@ Please ensure that you are using a supported version of PHP for the version of p
 * **v4.0.0** supports PHP 7.0 to 7.2
 * **v4.1.0+** supports PHP 7.1 to 7.3 \(inclusive\). This change was made to ensure we support versions of PHP that are getting security fixes at the very least. See PHP maintainance schedules [here](https://www.php.net/supported-versions.php).
 
-### I’m getting a database connection error
+### I’m getting a database connection error.
 
  Please verify that you have created your MySQL database, know the correct credentials for the API to connect to it and that your “.env” file is created with the expected format and at the expected location
 
-### How do I upgrade Ushahidi 
+### How do I update Ushahidi?
```

Comment:
I'd keep the "upgrade" framing or go for "update or upgrade" but I may be wrong 😬 just a suggestion